products/caregivers/1010cg-mvp/Usability Study-Sign as Representative- February 2021/conversation-guide.md

```diff
@@ -99,15 +99,19 @@ Great. Now we're really ready to begin!
   - Do they live with the Veteran?
 
 ## Applying for benefits task with first representative option - 15 minutes
+/// REVIEWER COMMENTS: 
+1. Are you giving them instructions about what they are looking at? Where are you dropping them into the application process? 
```

Comment:
Dropping them on the "intro page" of the Caregiver form. I could certainly add some context around what the participants are looking at.